schema-next.yaml

```diff
@@ -46,6 +46,12 @@ versions:
         # https://github.com/open-telemetry/semantic-conventions/pull/1006
         - rename_metrics:
             messaging.publish.messages: messaging.client.published.messages
+        # https://github.com/open-telemetry/semantic-conventions/pull/1212
+        - rename_attributes:
+            attribute_map:
+              system.process.status: process.status
+            apply_to_metrics:
+              - system.processes.count
```

Comment:
Shouldn't we include all the other process metrics here?

Reply:
This attribute was only previously used on `system.processes.count`, so the `rename` in particular is only to that usage of the attribute. Its usage as a resource attribute is new, which means it doesn't get represented in this file (only renames not additive changes are in `schema-next`).